### DISCUSSION TITLE
Is there anyway we can modify the request and response?

### DISCUSSION THREAD

**Discussion by akshaytandroid** [mocklets/pluto#193] [Q&A]
As titled,
Is there anyway we can modify the request and response?

**Comment by srtvprateek** [comment]
From what I understood from the questions, you want a functionality like breakpoints in Charles Proxy.

For now Pluto does not support any such feature, due to usability issues in limited screen size on a mobile device.
But if you want to mock the API response, it is supported via Mocklets (https://mocklets.com) integration.

Also, please share if you have any specific usecase, ll definitely look into it.
